src/progression/ProgressionForm.ts

```diff
@@ -53,6 +52,10 @@ export class ProgressionForm {
                 if (item && item.id && item.type === "Item") {
                     if (item.pack) {
                         this.addFromCompendium(item, dropTarget).then(_ => this.render())
+                        if (item.pack == "dnd5e.classes") {

```

Comment:
You can add it within the `addClass` method. All the class checks are done by that time already